<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Приветствие</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/logo.png</normaloff>../images/logo.png</iconset>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Бейсенов Меирбек
ВТиП-402
Задание №7</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_subject">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Тема - Периоды жизни!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
